content/articles/geospatial_standards/index.Rmd

```diff
@@ -0,0 +1,90 @@
+
+## Why use open standards?
+
+- Open file standards ease collaboration, portability and compatibility between users, machines and applications.
+- Their (file) structure is fully documented.
+  - Consequently, scientists and programmers can build new software / packages and make innovations that use these standards, while maintaining interoperability with existing applications.
+  - And, it guarantees that your data will still be readable in a hundred years from now, independently of which IT corporation or software is dominant at that time...
+  
+Luckily, quite a list of open standards is available!
+Below, some powerful and widely-used single-file formats are introduced.
+Single-file data sources are readily amenable to exchange and publication.
+
+I see you can't wait to start practicing, so you can also head straight over to the [tutorial on vector formats](../../tutorials/spatial_standards_vector/) and the [tutorial on the GeoTIFF raster format](../../tutorials/spatial_standards_raster/)!
+
+## A few words on the GDAL library
+
+**[GDAL](https://gdal.org)** (Geospatial Data Abstraction Library) is by far the most used collection of open-source drivers for:
+
+- [a lot](https://gdal.org/drivers/raster/index.html) of raster formats;
+- [a lot](https://gdal.org/drivers/vector/index.html) of vector formats.
+
+In other words, it is the preferred workhorse for reading and writing many geospatial file formats, used in the background by [a lot](https://gdal.org/software_using_gdal.html#software-using-gdal) of geospatial applications.
+Using GDAL is the easiest way to conform to open standards.
+
+So, in R we use packages that use GDAL in the background, such as `rgdal`, `sp`, `sf`, `raster` and `stars`.
+
+## The GeoPackage file format
+
+- Its website is <https://www.geopackage.org>.
+- It is a standardized implementation of an SQLite database for geospatial data.
+Hence, a GeoPackage is a **binary** file (`filename.gpkg`).
+It shares this property with shapefiles, which however pose multiple limitations ^[
+Some problems with shapefiles are: they're not an open format, they consist of multiple files and they have restrictions regarding file size, column name length, number of columns and the feature types that can be accommodated.
+], 
+so the GeoPackage is a more than suitable replacement.
+- The GeoPackage can store one or _multiple_ **vector** layers (points, lines, polygons and related feature types).
+Besides vector data, it can also store **raster** data
+or extra standalone **tables**.
+- The GeoPackage standard is [maintained](https://www.opengeospatial.org/standards/geopackage) by the [Open Geospatial Consortium](https://www.opengeospatial.org/) (OGC), which stands out as a reference when it comes to open geospatial standards.
+
+## The GeoJSON file format
+
+- One [GeoJSON](https://tools.ietf.org/html/rfc7946) file (`filename.geojson`) contains _one_ **vector** layer.
```

Comment:
This must be added indeed, thanks! In case of multiple types of features it is a 'FeatureCollection'. Still, I believe this does not contradict the concept of 'one vector layer', what is stated here. One `sf` object corresponds to one 'layer', which can have multiple feature types:

``` r
st_layers("batbike_inbo.geojson")
#> Driver: GeoJSON 
#> Available layers:
#>     layer_name geometry_type features fields
#> 1 batbike_inbo                     11      7

batbike <- st_read("batbike_inbo.geojson")
#> Reading layer `batbike_inbo' from data source `/media/floris/DATA/ContinuLegen/batbike_inbo.geojson' using driver `GeoJSON'
#> Simple feature collection with 11 features and 7 fields
#> geometry type:  GEOMETRY
#> dimension:      XY
#> bbox:           xmin: 4.198113 ymin: 50.70852 xmax: 4.278424 ymax: 50.75257
#> epsg (SRID):    4326
#> proj4string:    +proj=longlat +datum=WGS84 +no_def

st_geometry_type(batbike)
#>  [1] LINESTRING POINT      POINT      POINT      POINT      POINT      POINT      POINT     
#>  [9] POINT      POINT      POINT     
#> 18 Levels: GEOMETRY POINT LINESTRING POLYGON MULTIPOINT MULTILINESTRING ... TRIANGLE
```

Reply:
Clarified by 19bb8e7